Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11.11-slim
+
+ENV PYTHONBUFFERED 1
```

Comment:
The environment variable `PYTHONBUFFERED` is incorrectly named. It should be `PYTHONUNBUFFERED` to ensure that Python output is sent straight to the terminal without being buffered.